app/src/main/java/com/example/mysimplecoindeck/api/CoinApi.kt

```diff
@@ -17,4 +19,10 @@ interface CoinApi {
         @Path("id")
         uuid: String
     ): Response<CoinResponse>
+
+    @GET("search-suggestions")
+    suspend fun getSearchSuggestions(
+        @Query("query")
+        searchQuery: String
+    ): Response<SearchResponse>
```

Comment:
I don't think `Response` is needed, this can just return `SearchResponse` (same for the other call)